docs/samples/policyTemplateSchema.yaml

```diff
@@ -0,0 +1,73 @@
+# All the values from this schema are of string type as they will
+# end up as key:value pairs in a ConfigMap.
+description: ACM PolicyGenerator schema
+properties:
+
+  # site specific config
+  site: # this is the name of the ConfigMap holding site specific configuration
```

Comment:
Based on the CaaS provision doc, it seems like this top level field represents both the configmap name and  cluster label.